### ISSUE TITLE
Using this library for vanishing

### ISSUE THREAD

**Issue by LoganDark** [NucleoidMC/DisguiseLib#27]
Is it possible to make a vanish mod using this library that completely hides the player (see #26) but doesn't replace them with something else?

I ask because vanishing is pretty similar to disguising, and this library already lives in that space, so it wouldn't be too much of a scope increase if you do choose to start supporting it.

**Comment by Patbox** [comment]
It's not really similar, it's pretty different and you can't really use it for that

**Comment by LoganDark** [comment]
> It's not really similar, it's pretty different and you can't really use it for that

Curious; why is it not similar? Because only certain players have the vanished entity hidden?

**Comment by samolego** [comment]
You could disguise player as an "empty item", but there are better ways to handle vanishing imho.

**Comment by LoganDark** [comment]
> You could disguise player as an "empty item", but there are better ways to handle vanishing imho.

Yeah, because people would still be able to see the empty item entity - it would still have a shadow iirc among other things.
